### PR TITLE
Alex/gh UI admin

### PR DESCRIPTION
**Pull request by whiteadam2**

### CODE REVIEW

File: apps/monitor-client/lib/components/layout/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <Grid container spacing={3}>
+            <Grid item xs={12} md={8} lg={9}>
+                {children}
+            </Grid>
+        </Grid>
+    );
+}
+
+Content.propTypes = {
+    '': PropTypes.func,
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/layout/Content.tsx:17]: '': PropTypes.func, what is this?

File: apps/monitor-client/lib/components/layout/Copyright.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/layout/Copyright.tsx:1]: copyright should be in components but not in layout

File: apps/monitor-client/lib/app/AppContainer.tsx [hunk 3]
```diff
@@ -1,54 +1,12 @@
-import React, { useContext, useState, useEffect } from 'react'
-import { AppContext, IAppContext } from '../context';
+import React from 'react';
+
 import { Login } from '../features/auth/Login';
+import { HomePage } from '../features/HomePage';
 
 export function AppContainer() {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/app/AppContainer.tsx:6]: the component should contain high level routing

File: apps/monitor-client/lib/components/UsersTable.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import React, { useContext, useState, useEffect } from 'react';
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/UsersTable.tsx:1]: please move file to /features/users/components

File: apps/monitor-client/lib/components/layout/Content.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from '@mui/material';
+
+export function Content({ children }: any) {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/layout/Content.tsx:6]: use render props

File: apps/monitor-client/lib/components/layout/Layout.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+    return (
+        <ThemeProvider theme={mdTheme}>
+            <Box sx={{ display: 'flex' }}>
+                <CssBaseline />
+
+                {renderHeader({ drawerWidth, toggleDrawer, open })}
+                {renderSidebar({ drawerWidth, toggleDrawer, open })}
+
+                <Box
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/layout/Layout.tsx:35]: move this Box to the content layout

File: apps/monitor-client/lib/components/layout/ListItems.tsx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import ListSubheader from '@mui/material/ListSubheader';
+import ViewListIcon from '@mui/icons-material/ViewList';
+import PeopleIcon from '@mui/icons-material/People';
+import WebhookIcon from '@mui/icons-material/Webhook';
+import AssignmentIcon from '@mui/icons-material/Assignment';
+
+export function MainListItems() {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/components/layout/ListItems.tsx:15]: move to lib/components/

File: apps/monitor-client/lib/features/CollapsedView.tsx [hunk 8]
```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { AppContext, IAppContext } from '../context';
+import TableCollapsible from '../components/UsersTable';
+
+export function CollapsedView() {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/CollapsedView.tsx:5]: CollapsedView of what?
this is users feature? or this is reusable component?
in first case move it to the /features/users/components/
in second lib/components

File: apps/monitor-client/lib/features/HomePage.tsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+import { Layout } from '../components/layout/Layout';
+import { Sidebar } from '../components/layout/Sidebar';
+import { Header } from '../components/layout/Header';
+import { Content } from '../components/layout/Content';
+
+import { ServerView } from './ServerView';
+import { UsersView } from './UsersView';
+import { SessionsView } from './SessionsView';
+
+export function HomePage({}) {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/HomePage.tsx:14]: lib/features/homepage/components

also, add to all folders index.js with export components within this folder

File: apps/monitor-client/lib/features/ServerView.tsx [hunk 10]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/ServerView.tsx:2]: lib/features/server/components

File: apps/monitor-client/lib/components/layout/Content.tsx (hunk 1 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/components/layout/Content.tsx]: my template from zorko

File: apps/monitor-client/lib/app/AppContainer.tsx (hunk 3 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/app/AppContainer.tsx:6]: done, but route "/login" wont work

File: apps/monitor-client/lib/components/layout/Layout.tsx (hunk 6 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/components/layout/Layout.tsx:34]: I've removed content layout

File: apps/monitor-client/lib/components/layout/ListItems.tsx (hunk 7 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/components/layout/ListItems.tsx]: moved to /lib/components/layout/sidebar/

File: apps/monitor-client/lib/features/HomePage.tsx (hunk 9 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/features/HomePage.tsx]: I think making structure like that is complicated, I don't expect a lot of component inside particular feature

**Review by yzhbankov — APPROVED** [review]

**Pull request merged**